client/src/components/FieldHolder/FieldHolder.js

```diff
@@ -1,5 +1,5 @@
 import React, { Component } from 'react';
-import { FormGroup, InputGroup, InputGroupAddon, Label } from 'reactstrap';
+import { FormGroup, InputGroup, InputGroupText, Label } from 'reactstrap';
```

Comment:
https://reactstrap.github.io/?path=/story/home-upgrading--page

> `InputGroupAddon` - This in no longer needed in Bootstrap 5, you can now add Buttons, Inputs directly to InputGroups in the order you wish.

Note that `InputGroupAddon` used to implicitly add a `InputGroupText` if it was a string being added, so I've opted to keep that behaviour here.